src/main/java/peppa/Peppa.java

```diff
@@ -0,0 +1,56 @@
+package peppa;
+
+import peppa.commands.Command;
+
+/**
+ * Represents the chatbot application.
+ */
+public class Peppa {
+    public static final String FILE_PATH = "data/todo.txt";
```

Comment:
I like how you made the file path a constant name so as to promote encapsulation. 
I would incorporate this idea as well!

Just one quick note, since you are declaring FILE_PATH and using in the same class, would it be better to give it private instead of public as modifier?

```suggestion
    private static final String FILE_PATH = "data/todo.txt";
```